client/src/components/Input/Checkbox.js

```diff
@@ -3,12 +3,12 @@ import styled from "styled-components";
 const StyledCheckbox = styled(Checkbox)`
   .ant-checkbox-inner {
     border-color: #646464;
-    width: 2rem;
-    height: 2rem;
+    width: ${(props) => props.size || "2rem"};
+    height: ${(props) => props.size || "2rem"};
   }
   .ant-checkbox-checked .ant-checkbox-inner {
     background-color: ${(props) => props.color || "#6076ef"};
-    border-color: #939393;
+    border-color: ${(props) => props.color || "939393"};
```

Comment:
Update color to DARK GRAY. Not sure the background color with the constants list.

Reply:
Yeah, I went with the exact color from Zeplin which wasn't in our theme. Not sure if it's worth it to add this exact color to the theme or just go with the colors.DARK_GRAY?